ambry-cloud/src/main/java/com/github/ambry/cloud/azure/AzureCloudConfig.java

```diff
@@ -96,6 +101,24 @@
   @Config(COSMOS_DELETED_CONTAINER_COLLECTION_LINK)
   public final String cosmosDeletedContainerCollectionLink;
```

Comment:
Sorry for the late feedback, upon reading this, it seems like cosmosCollectionLink/cosmosDeletedContainerCollectionLink aren't used in the codebase anymore, so even thought they are present here, they won't be honored by this version of the code.

I think we can go ahead and remove them from linkedin/ambry. Configurations within LI will have to set both of these to be compatible with older versions of the code, but it would keep the newest code slightly cleaner and we could clean up the configs after we roll this out.

Another more complicated approach would be to keep the collection link configs as is, and just parse out the DB and container names from them. Benefit of this would be that a config migration isn't needed, but it leads to the java code being messier, imo.

Reply:
Sure! Removed the old configs for cosmos collections from here. Will keep them in LI until the changes are rolled out.